drkafka/src/main/java/com/pinterest/doctorkafka/modules/plugins/kafka/NoStatsBrokerMonitor.java

```diff
@@ -0,0 +1,37 @@
+package com.pinterest.doctorkafka.modules.plugins.kafka;
+
+import com.pinterest.doctorkafka.modules.context.KafkaClusterContext;
+import com.pinterest.doctorkafka.notification.Email;
+
+import kafka.cluster.Broker;
+import scala.collection.Seq;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NoStatsBrokerMonitor extends KafkaPlugin {
+  public void evaluate(KafkaClusterContext ctx) {
+    // check if there is any broker that do not have stats.
+    List<Broker> noStatsBrokers = getNoStatsBrokers(ctx);
+    if (!noStatsBrokers.isEmpty()) {
+      ctx.setNoStatsBrokers(noStatsBrokers);
+      Email.alertOnNoStatsBrokers(ctx.getAlertEmails(), ctx.getClusterName(), noStatsBrokers);
```

Comment:
Yes, operations/actions are to be moved out of these monitoring plugins. This will be done in future PRs, this PR only ports original KafkaClusterManager code directly with minimal modification.